components/automate-ui/src/app/entities/layout/layout.model.ts

```diff
@@ -1,21 +1,38 @@
+import { Observable } from 'rxjs';
+
+// Sidebar {
```

Comment:
still needed?

Reply:
@bcmdarroch still needed but noticed I needed to rename it and fix a few other comment examples 👍🏻